scripts/quests/otherAreas/For_the_Birds.lua

```diff
@@ -0,0 +1,211 @@
+-----------------------------------
+-- For the Birds
+-----------------------------------
+-- Log ID: 4, Quest ID: 104
+-- Koblakiq          !pos -64 21 -117 11
+-- Daa Bola the Seer !pos -157 -17 193 151
+-- GeFhu Yagudoeye   !pos -91 -3 -127 147
+-----------------------------------
+local oztrojaID  = zones[xi.zone.CASTLE_OZTROJA]
+local beadeauxID = zones[xi.zone.BEADEAUX]
+-----------------------------------
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.FOR_THE_BIRDS)
+
+quest.reward =
+{
+    item = xi.item.JAGUAR_MANTLE,
+}
+
+local mobListTable =
+{
+    beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET,
+    beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET + 1,
+    beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET + 2,
+    beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET + 3,
+}
+
+local function AllDead(mobs)
+    for _, mobID in ipairs(mobs) do
+        local mob = GetMobByID(mobID)
+        if not mob:isDead() then
+            return false
+        end
+    end
+
+    return true
+end
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+            player:hasCompletedQuest(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.MISSIONARY_MOBLIN) and
+            not quest:getMustZone(player)
+        end,
+
+        [xi.zone.OLDTON_MOVALPOLOS] =
+        {
+            ['Koblakiq'] = quest:progressEvent(14, { [1] = xi.item.ARNICA_ROOT }),
+
+            onEventFinish =
+            {
+                [14] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:begin(player)
+                    end
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_ACCEPTED
+        end,
+
+        [xi.zone.OLDTON_MOVALPOLOS] =
+        {
+            ['Koblakiq'] =
+            {
+                onTrigger = function(player, npc)
+                    local prog = quest:getVar(player, 'Prog')
```

Comment:
Can we set a standard here? Most examples use `questProgress`

Reply:
Done ✅